Localize/lang/strings.json

```diff
@@ -2258,7 +2258,6 @@
   "_ljAOR6.comment": "Label for description of custom body Function",
   "_lkgjxD.comment": "Required string parameter for end time",
   "_lkmLHk.comment": "Toggle to use schema editor",
```

Comment:
You can merge from latest `main` and this file will be removed as this change has been completed in another PR.